service/service.go

```diff
@@ -157,9 +157,9 @@ func (s *Service) GetObject(id []byte) (*GetObjectResult, error) {
 	if err != nil {
 		return nil, err
 	}
-	v := make([]float64, len(vector))
+	v := make([]float32, len(vector))
 	for i, e := range vector {
```

Comment:
if you are using gongt.GetStrictVector, the vector type is already []float32 you don't need loop casting